Source/swiftlint/Helpers/LintableFilesVisitor.swift

```diff
@@ -137,13 +137,14 @@ struct LintableFilesVisitor {
             return .success(.buildLog(compilerInvocations: compilerInvocations))
         } else if !options.compileCommands.isEmpty {
             let path = options.compileCommands
-            guard let compileCommands = self.loadCompileCommands(path) else {
+            switch self.loadCompileCommands(path) {
+            case .success(let compileCommands):
+                return .success(.compilationDatabase(compileCommands: compileCommands))
+            case .failure(let error):
                 return .failure(
-                    .usageError(description: "Could not read compilation database at path: '\(path)'")
+                    .usageError(description: "Could not read compilation database at path: '\(path)' \(error.description)")
```

Comment:
This line is triggering a line length violation because it's 123 characters long, beyond the 120 character limit.